5.5/examples/replica/persistent_nfs_volume.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
not to mention it crosses the boundary between what a user trying to run this sample needs to do, and what an admin setting up an openshift environment needs to do.